help.txxt

```diff
@@ -0,0 +1 @@
+how do I make class? please help me please
```

Comment:
no classes in assembly

Reply:
what's assembly? I need help, project due yestdery it in python